html/cross-origin-embedder-policy/resources/nothing.txt.headers

```diff
@@ -0,0 +1 @@
+Cross-Origin-Resource-Policy: cross-origin
```

Comment:
@mikewest https://mikewest.github.io/corpp/ contains both "cross-site" and "cross-origin" and I don't know which is correct. Can you update the document?

Reply:
https://github.com/mikewest/corpp/pull/1